app/experimenter/experiments/api/v2/serializers.py

```diff
@@ -254,8 +254,6 @@ def update(self, instance, validated_data):
                 if removed_ids:
                     ExperimentVariant.objects.filter(id__in=removed_ids).delete()
 
-            self.update_changelog(instance, validated_data)
```

Comment:
This is a little bit wonky, taking it out of base and then adding it back into every child?  We should be able to keep it in here and not have to add it to every subclass?